### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by wrkode**
document --privileged flag requirement for Elemental builds

### CODE REVIEW

**Comment by ipetrov117** [comment]
@wrkode, can you also remove the [package installation note](https://github.com/wrkode/edge-image-builder/blob/patch-1/README.md?plain=1#L80). It is obsolete now.

**Comment by wrkode** [comment]
> @wrkode, can you also remove the [package installation note](https://github.com/wrkode/edge-image-builder/blob/patch-1/README.md?plain=1#L80). It is obsolete now.

removed and ready for review

**Review by ipetrov117 — APPROVED** [review]
Changes look good.

cc @atanasdinov  for a second set of eyes.

**Review by jdob — APPROVED** [review]
Good pickup, thanks :)

**Review by jdob — CHANGES_REQUESTED** [review]
I think we can skip this in favor of what's in #192

**Comment by wrkode** [comment]
> I think we can skip this in favor of what's in #192

We shouldn't skip this. these are two independent documents.
The information for the build must result in a successful execution. this change allows that to occur. I'm ok for a generic README, but must be correct.

**Comment by wrkode** [comment]
> I looked at this before noticing #192. Feels redundant to have in both places, and this bleeds specific implementation stuff into an otherwise generic README.

the documents are independent.

File: README.md [hunk 1]
```diff
@@ -67,8 +77,7 @@ podman run --rm -it \
 * `--validate` - If specified, the specified image definition and configuration directory will be checked to ensure
   the build can proceed, however the image will not actually be built.
 
-> **_NOTE:_** When specifying packages for installation, you need to also pass the [`--privileged`](https://docs.podman.io/en/latest/markdown/podman-run.1.html#privileged) option as well. For more information on why this is needed, see the [Package resolution design](./docs/design/pkg-resolution.md#running-the-eib-container) documentation.
```
Comment (atanasdinov) [inline README.md:70]: Please consider rewording this along the lines of:
```
Build process which involves package resolution must include the [`--privileged`](https://docs.podman.io/en/latest/markdown/podman-run.1.html#privileged) flag.
Package resolution will be automatically performed when requesting package installation or configuring components which require it (e.g. Elemental, Kubernetes SELinux, etc.)
```

File: README.md [hunk 2]
```diff
@@ -53,6 +53,16 @@ podman run --rm -it \
 --build-dir /eib/_build
 ```
 
+**NOTE:**
+When specifying packages for installation or doing elemental builds you need to also pass the [`--privileged`](https://docs.podman.io/en/latest/markdown/podman-run.1.html#privileged) flag as well. For more information on why this is needed, see the [Package resolution design](./docs/design/pkg-resolution.md#running-the-eib-container) documentation.
+```shell
+podman run --rm --privileged -it \
```
Comment (atanasdinov) [inline README.md:59]: This example introduces redundancy.

**Review by atanasdinov — CHANGES_REQUESTED** [review]
Hi, @wrkode. Thanks for the contribution!

I share @jdob's opinion in that this change would introduce redundancy and that component specific documentation belongs to the [Building images](https://github.com/suse-edge/edge-image-builder/blob/main/docs/building-images.md) guide.

We're in the process of actively building up our documentation as well as looking forward to introducing a simpler way of running EIB in the first place (with https://github.com/suse-edge/edge-image-builder/issues/72).

Until then keeping the main README as clean as possible is in my opinion the best way forward.

If you're keen on contributing this, please consider the suggestions above.

File: README.md (hunk 1 above)
Comment (wrkode) [reply README.md:70]: agreed, this seems a sensible comment

**Review by wrkode — COMMENTED** [review]
looks good

**Review by jdob — APPROVED** [review]

**Comment by atanasdinov** [comment]
@wrkode do we want to proceed with this? I see that you've agreed with the suggestions but there's no commit addressing them yet.

**Comment by wrkode** [comment]
> @wrkode do we want to proceed with this? I see that you've agreed with the suggestions but there's no commit addressing them yet.

missed the commit. yes LGTM and committing now

**Review by wrkode — COMMENTED** [review]
LGTM

**Comment by wrkode** [comment]
@atanasdinov , I've committed the suggest changes

**Review by atanasdinov — APPROVED** [review]
Thanks!

**Pull request merged**